chia/wallet/coin_selection.py

```diff
@@ -97,7 +102,7 @@ async def select_coins(
                 coin_set = {greater_coin}
         return coin_set
     else:
-        # if smaller_coin_sum == amount and len(smaller_coins) >= max_num_coins.
+        # if smaller_coin_sum == amount and len(smaller_coins) >= max_num_coins, or amount == 0.

```

Comment:
This comment seems inconsistent with the line it's trying to reference.  I can't tell if it's incorrect, but it's different.

Reply:
yeah ill make it clearer